components/date-picker/date-picker.jsx

```diff
@@ -59,6 +60,10 @@ const propTypes = {
 		PropTypes.object,
 		PropTypes.string,
 	]),
+	/**
+	 * Establishes directional context for component. Defaults to left-to-right.
+	 */
+	direction: PropTypes.oneOf([DIRECTIONS.LTR, DIRECTIONS.RTL]),
```

Comment:
FYI: I’m not sure what direction we need to go, but @andyouyang22 and @minevskiy are talking about context only usage over on Dropdown component.

Reply:
@hajiz , DatePicker itself doesn't need to know about direction. Its children do. The whole point of context is to avoid passing props through the hierarchy of components.
Please undo changes in this file and in `date-picker/index.jsx`, `calendar-wrapper.jsx`, `calendar.jsx`, and instead wrap the export of `date-picker/private/day.jsx` with `LanguageDirection`.

Reply:
@interactivellama , I didn't mean to suggest to only use the context. I meant that, if a component doesn't need to change its look or behavior based on direction, the it needs neither `direction` prop nor `LanguageDirection` HOC.

Reply:
Gotcha, I was wondering about this too. I'll remove the passing of params and switch to wrapping the day component in language direction.

Reply:
Thank you for clarifying. I stepped into the middle without understanding.

Reply:
Addressed in 4553397

Thanks Stephan for catching it and Ivan for pointing out the good practice.